docs/Actions/sanitize-html.md

```diff
@@ -0,0 +1,118 @@
+---
+id: sanitize-html
+title: Sanitize HTML
+sidebar_label: Sanitize HTML
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `HTML`
+
+This action recieves HTML input and repairs broken HTML tag syntax, removes specifed HTML tags and attributes, sets the "rel" attribute to "nofollow" on anchor tags and removes scripts, based on the settings that are selected and set by the user.
+
+## `Typical Use Cases`
+
+- Apply to the input of a field in a form before the data is saved to your database.
+- Apply to HTML recieved via an API before it is saved to your database.
+- Apply to the content of an incoming email before the content is saved to your database.
+
+## `Don't use it to`
+
+- Sanitize multiple inputs at once. Use a separate action for each input.
+- To prevent SQL injection. Use Bind Tokens in your [Run SQL Query](/docs/Actions/run-sql) action instead.
+
+## `Related Actions`
+
+| Action Name                                     | Description                                                                        |
+| ----------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Generate PDF](/docs/Actions/generate-pdf.md)   | Uses wkhtmltopdf open source to generate PDF from HTML template.       |
```

Comment:
I think it would be useful to describe why it would be related. For example, one might one to not show some tags in the PDF. Same for the rest of related actions.

Reply:
I expanded descriptions in latest commit